core/src/main/java/com/oath/oak/NativeAllocator/Block.java

```diff
@@ -102,7 +102,7 @@ public ByteBuffer getReadOnlyBufferForThread(int position, int length) {
         bb.limit(position + length);
         bb.position(position);
         // on purpose not creating a ByteBuffer slice() here,
-        // slice() will be used only per demand when buffer is passed to the serializer
+        // slice() will be used only per demand when buffer is passed outside Oak
```

Comment:
Comment unclear

Reply:
Please read two comment lines together. Is it still unclear?